test/WebSites/WebApiCompatShimWebSite/WebApiCompatShimWebSite.csproj

```diff
@@ -1,9 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(PackageTargetFallback);portable-net451+win8</PackageTargetFallback>
+    <TargetFrameworks>$(StandardTestWebsiteTfms)</TargetFrameworks>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'netcoreapp2.0'  ">$(PackageTargetFallback);portable-net451+win8</PackageTargetFallback>
```

Comment:
Any reason this can be defined alongside StandardTestWebsiteTfms in DB.props?

Reply:
Seemed counter intuitive to define a property that applies to the minority of projects (3 in all). Plus we're supposed to remove it. Any day now!